File: Print the Element Greater Than the Previous Element/print_element_greater_than_previous.c
#include <stdio.h>
int main() {
    int n;
    printf("Enter the number of elements: ");
    scanf("%d", &n);
    int a[n];
    for (int i = 0; i < n; i++) {
        printf("Enter element %d: ", i + 1);
        scanf("%d", &a[i]);
    }
    printf("Elements greater than the previous element:\n");
    for (int i = 1; i < n; i++) {
        if (a[i] > a[i - 1]) {
            printf("%d\n", a[i]);
        }
    }
    return 0;
}

File: Print the Element Greater Than the Previous Element/print_element_greater_than_previous.py
n = int(input("Enter the number of elements: "))
a = []
for i in range(n):
    a.append(int(input(f"Enter element {i + 1}: ")))
print("Elements greater than the previous element:")
for i in range(1, n):
    if a[i] > a[i - 1]:
        print(a[i])
